molecule/default/vars-ubuntu18.yml

```diff
@@ -1,4 +1,4 @@
-MOLECULE_IT_IMAGE_ID: ami-08bd6fc5c8daf3309 # ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-20230517
+MOLECULE_IT_IMAGE_ID: ami-02076d50a3b3930d0 # ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server
```

Comment:
timestamp was cool to see at glance how old it is

Reply:
same as above